app/managers/img_manager.py

```diff
@@ -0,0 +1,50 @@
+from io import BytesIO
+from zipfile import ZipFile
+from tempfile import TemporaryDirectory
+from os import listdir, path
+
+from imgprep import Preprocessor
+
+
+def read_archive(data):
+    """Read archive of images and send it to application"""
+    _read_archive(data)
+    return
+
+
+def _read_archive(data):
+    archive = BytesIO(data)
+    zip_file = ZipFile(archive, "r")
+    with TemporaryDirectory() as td:
+        zip_file.extractall(td)
+        for file_name in listdir(td):
+            _read_image(path.join(td, file_name))
+
+
+def _read_image(file_name, digit=None):
```

Comment:
I add option "digit" for future. If we need to implement loading images as single files, we need to call _read_image method with taken digit from user